src/main/java/gov/nasa/pds/tools/validate/content/table/TableContentProblem.java

```diff
@@ -32,7 +32,7 @@ public class TableContentProblem extends ContentProblem {
   private Integer table;
   
   /** The record number associated with the problem. */
-  private Integer record;
+  private long record;
```

Comment:
What's the reason going from object Integer to primitive long and not object Long?

Reply:
See above.

Reply:
Well at least Java does "auto-boxing" so it'll probably never be an issue; let's just ignore it!

Reply:
Roger roger